memebot/command.py

```diff
@@ -20,17 +20,30 @@ def help_string(self) -> str:
         pass
 
 
+def create_embed(*, commands: Iterable[Tuple[CommandExecutor, Iterable[str]]], **kwargs) -> discord.Embed:
```

Comment:
Is there a way to simplify the parameters of the `create_embed` function? I mean, `Iterable[Tuple[CommandExecutor, Iterable[str]]]`? Seems pretty circle jerk-y to me. Also, why is the executor the one with the help string? You would think commands have help strings and then you would literally just pass this thing a list of commands.

Reply:
Yeah I should just make an object to contain those. I guess right now, `Command` refers to an instance of a user trying to use a command, whereas the `CommandExecutor` is the thing that actually runs it. Maybe I should rename to make that less confusing.